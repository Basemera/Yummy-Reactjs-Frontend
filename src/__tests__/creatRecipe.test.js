import React from 'react';
import { shallow, enzyme } from 'enzyme';
import { MemoryRouter } from 'react-router-dom';
import toJson, { shallowToJson } from 'enzyme-to-json';
import sinon from 'sinon';

import CreateRecipe from '../components/recipes/CreateRecipes';

describe('Signup component', () => {
  const wrapper = shallow(<CreateRecipe />);

  it('renders properly', () => {
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
