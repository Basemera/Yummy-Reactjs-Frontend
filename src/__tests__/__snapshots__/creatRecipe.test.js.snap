// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreateRecipe component renders properly 1`] = `
<div>
  <div
    id="recipesbar"
  >
    <div
      className="container text-center"
    >
      <div
        className="row"
      >
        <div
          className="col-xs-8  col-sm-6 offset-xs-2 offset-sm-3 col-md-4 offset-md-4"
        >
          <h2>
            Create Your Recipes
          </h2>
          <form
            onSubmit={[Function]}
          >
            <div
              className="input-group"
            >
              <span
                className="input-group-addon"
              >
                <i
                  className="glyphicon glyphicon-pencil"
                />
              </span>
              <input
                className="form-control"
                name="title"
                onChange={[Function]}
                placeholder="recipe title"
                type="text"
                value=""
              />
            </div>
            <br />
            <div
              className="input-group"
            >
              <span
                className="input-group-addon"
              >
                <i
                  className="glyphicon glyphicon-list-alt"
                />
              </span>
              <textarea
                className="form-control description"
                cols="50"
                name="description"
                onChange={[Function]}
                placeholder="description or instructions"
                rows="4"
                value=""
              />
            </div>
            <br />
            <input
              className="btn btn-primary"
              type="submit"
              value="Add Recipe"
            />
          </form>
        </div>
      </div>
    </div>
    <br />
  </div>
  <div>
    <div />
  </div>
</div>
`;
